templates/editSchedule.html

```diff
@@ -11,11 +11,11 @@
           {{> afQuickField name="calendarEndMinutes" }}
         {{/autoForm}}
 
-        <h3>Unscheduled Rounds:</h3>
+        <h3>Drag to schedule ⇒</h3>
         <div id="unscheduled-rounds">
```

Comment:
I haven't tried this out yet, but is this id still accurate (unscheduled-rounds)?